ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.hbs

```diff
@@ -29,4 +29,8 @@
     </div>
 </div>
 
+{{#is type 'custom'}}
+    <span class="interaction interaction-custom"></span>
```

Comment:
This should be a div.  Spans are not meant to house block level elements.

Reply:
Done